massgov/mayflower
# DP-9867: Hotfix for pagination in URL

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description

Changes url-based pagination to use the `_page` query parameter instead of the `page` parameter.  

Can be tested on the drupal side by updating 178 of `composer.json` to:
```
"massgov/mayflower-artifacts": "dev-patternlab/hotfix-DP-5329",
```
Then running `composer udpate massgov/mayflower-artifacts`.  I'm not sure why, but `composer require` isn't working for this branch.  

## Related Issue / Ticket

- [JIRA issue](https://jira.mass.gov/browse/DP-9867?focusedCommentId=247565&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-247565)

## Steps to Test


1. Visit http://mass.local/massachusetts-hunter-education-program
2. Click the "See all 10 items"
3. Once the listing page loads, reload the page and verify that you don't get a 404.

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
